lrpar/src/lib/ctbuilder.rs

```diff
@@ -618,3 +618,67 @@ pub fn _reconstitute<'a, StorageT: Deserialize<'a> + Hash + PrimInt + Unsigned>(
     let stable = deserialize(stable_buf).unwrap();
     (grm, sgraph, stable)
 }
+
+#[cfg(test)]
+mod test {
+
+    extern crate temp_testdir;
+    use std::path::PathBuf;
+    use std::fs::File;
+    use std::io::Write;
+
+    use self::temp_testdir::TempDir;
+    use super::{CTParserBuilder, ActionKind, CTConflictsError};
+
+    #[test]
+    fn test_conflicts() {
+
+        let temp = TempDir::default();
+        let mut file_path = PathBuf::from(temp.as_ref());
+        file_path.push("grm.y");
+        let mut f = File::create(file_path.clone()).unwrap();
+        let _ = f.write_all("%start A
+%%
+A : 'a' 'b' | B 'b';
+B : 'a' | C;
+C : 'a';".as_bytes());
+
+        match CTParserBuilder::<u8>::new()
```

Comment:
Once you merge in master you can (and probably will have to) remove the `<u8>` annotation here.